src/style.css

```diff
@@ -1 +1,75 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Book.ttf');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('fonts/Avenir-Heavy.ttf');
+  font-weight: bold;
+}
+
+body {
+  padding: 0;
+}
+
+.search-bar {
+  margin: 20px 0;
+}
+
+.search-bar__input {
+  box-sizing: border-box;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  color: #3d4e61;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background-image: url(./images/Search.svg);
+  background-repeat: no-repeat;
+  background-size: 19px;
+  background-position: 25px;
+  width: 100%;
+  height: 70px;
+  padding-left: 62px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+}
```

Comment:
Avoid fixing the container size like setting a height (line 35). Let the content size dictate it, to avoid overflow or accidental scroll bars. Consider removing the `height` property and using `padding` to control the size.